pkg/daemon/update.go

```diff
@@ -124,12 +124,12 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) error {
 	reconcilableError := dn.reconcilable(oldConfig, newConfig)
 
 	if reconcilableError != nil {
```

Comment:
Can we then say `if reconcilableError {` now?

Reply:
nope, it's still an error (your case it's only valid for `bool` in Go)

Reply:
ahh TIL thank you!!